docs/DeveloperGuide.adoc

```diff
@@ -279,87 +279,362 @@ Refer to the guide <<DevOps#, here>>.
+* 1a1. V outputs error informing user of invalid characters
+* 1a2. V terminates the command
++
+Use case ends
+
+[none]
+* 1b. V detects missing fields in contact entry
+* 1b1. V outputs error informing user of missing contact entry
+* 1b2. V terminates the command
++
+Use case ends
+
+[none]
+* 1c. V is unable to detect a valid keyword
+* 1c1. V outputs error informing user of invalid keyword
+* 1c2. V terminates the command
++
+Use case ends
+
+[discrete]
+=== UC06 - Adding Travel Entry into Journal in a Trip
+
+**Precondition: **User is already in the Journal page of Trip
+
+**Guarantee: **A new travel entry will be added to the Travel Journal
+
+*MSS*
+
+1. User enters details of the travel entry
+2. V confirms successful entry and displays new entry in the journal
++
+Use case ends
+
+*Extensions*
+
+[none]
+* 1a. V detects invalid characters into travel entry
+* 1a1. V outputs error informing user of invalid characters
+* 1a2. V terminates the command
++
+Use case ends
+
+[none]
+* 1b. V detects missing fields in travel entry
+* 1b1. V outputs error informing user of missing travel entry
+* 1b2. V terminates the command
++
+Use case ends
+
+[none]
+* 1c. V is unable to detect a valid keyword
+* 1c1. V outputs error informing user of invalid keyword
+* 1c2. V terminates the command
++
+Use case ends
 
 [discrete]
-=== Use case: Delete person
+=== UC07 - Adding Activities into Trip Itinerary
+
+**Precondition: **User is already in the Itinerary page of the Trip
+**Guarantee: **A new activity will be added to the Trip Itinerary
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. User enters activity name, date, time and location
+2. V confirms successful entry and displays updated itinerary
 +
-Use case ends.
+Use case ends
 
 *Extensions*
 
 [none]
-* 2a. The list is empty.
+* 1a. V detects invalid characters into activity entry
+* 1a1. V outputs error informing user of invalid characters
+* 1a2. V terminates the command
 +
-Use case ends.
+Use case ends
 
-* 3a. The given index is invalid.
+[none]
+* 1b. V detects missing fields in activity entry
+* 1b1. V outputs error informing user of missing activity entry
+* 1b2. V terminates the command
 +
+Use case ends
+
 [none]
-** 3a1. AddressBook shows an error message.
+* 1c. V is unable to detect a valid keyword
+* 1c1. V outputs error informing user of invalid keyword
+* 1c2. V terminates the command
 +
-Use case resumes at step 2.
+Use case ends
+
+[discrete]
+=== UC08 - Deleting a Contact in the Contact List of aTrip
+
+*Precondition:* User is in Contact List page of the Trip
 
-_{More to be added}_
+*Guarantee:* The contact in the Contact List will be deleted
+
+*MSS*
+
+[none]
+1. User requests to delete a contact in the contact list at the specified index
+2. V deletes the contact at the specified index and display the new contact list
+3. Use case ends
```

Comment:
Probably "use case ends" shouldn't b part of the list here, to standardise with previous use cases